app/javascript/components/widget-controller.jsx

```diff
@@ -72,6 +73,9 @@ export const WidgetController = ({ client, cityName }) => {
           console.error(error);
           return <DisconnectedDisplay cityName={cityName} error={error} />;
         }
+        if (data.location.widgets.enabled === []) {
```

Comment:
:thinking: we probably should access this path safely in case data is null / empty. Have a look at ramdajs `path` / `pathOr`

Reply:
this is a good idea